package.json

```diff
@@ -12,7 +12,7 @@
     "check": "yarn lint && yarn prettier-check && yarn test",
     "check-ci": "yarn run check",
     "format": "yarn run lint-fix && yarn run prettier-fix",
-    "lint": "eslint '*.js' 'examples/**/*.js' src scripts 'docs/*.js' docs/src",
+    "lint": "eslint '*.js' 'examples/**/*.js' src scripts",
```

Comment:
@jhumbug @carlos-kelly -- So the awkward thing we have right now is the play between root and docs eslints. The doc eslint only covers `*.js` which means that now we're missing coverage for `docs/src/`. Some thoughts:

- Do we want to have two lints? Or just pull out of docs and do all in root
- If we do have two lints, we need to make sure `docs/src` is covered as it isn't right now in this PR.

Reply:
As an FYI - The reason I made these changes stems from the separate workflows approach.  If you try to lint on the docs workflow, you only have the docs node_modules to work with. But the linter reaches up a directory and uses the eslintrc from the root.  That root eslintrc specifies the eslint-typescript parser and some eslint plugins that aren't in the docs' package json, so the lint doesn't work for docs since the root node_modules aren't installed.

I believe if we want to keep separate workflows and pull the lint out of docs and do it all in root, we'll have to install the root node_modules as part of the docs workflow in order to lint them when there are docs changes.  Should be doable.

I went ahead added 'src' to the docs linter so that should be covered if we decide to have two linters.

Reply:
Oh, that makes sense! You know, one thing that might be good for harmonizing is doing and extends in the configuration of the root eslint  configuration instead of a completely new one

Reply:
I went ahead and made the change so that there's only one linter again, while keeping separate gh workflows.  All I had to do was add the install of the root node_modules in the docs workflow.  Adds a few seconds to the ci/cd, but reduces complexity, so I think this is a good trade off.